src/components/DataViewFilterText/DataViewFilterText.tsx

```diff
@@ -119,8 +118,23 @@ function DataViewFilterText(props: DataViewFilterTextProps) {
 	// 	throwOnInvalid : true
 	// });
 
+	if (props.args.comparisonDefault && validComparisons.find( validComparison => validComparison.value === props.args.comparisonDefault) === undefined)
+		throw new Error("The selected comparison is not a valid comparison");
+
+	const getComparison = () => {
+		if (props.args && props.args.comparisons) {
+			if (props.data.comparison) {
+				return props.data.comparison;
+			} else if (props.args.comparisonDefault) {

```

Comment:
There's one small thing missing here... the ifs say: If I have comparisons, then if I have a comparison selected by the user, then return it, if I don't and I have a comparisonDefault then return it... but this is not contemplating that the comparisons that developers pass might not contain comparisonDefault. A practical example:
props.args.comparisons = ['contains', 'not_contains'],
props.args.comparisonDefault = 'equals'

This means that our filterText component will "select" the equals comparison but it isn't even present in the list of available comparisons, so it will break. I suggest doing something like this in line 128 else if (props.args.comparisonDefault && props.args.comparisons.includes(props.args.comparisonDefault))
After that, if that's not the case then default to "equals"